recipe/meta.yaml

```diff
@@ -1,48 +1,32 @@
-{% set gh_org = "jupyter" %}
-{% set gh_repo = "nbgrader" %}
 {% set version = "v0.5.1" %}
-{% set sha256 = "971a37b58a8f1a26e7b17eacb091879b5f1646f5097b717626436a266a178d33" %}
 
 package:
-  name: {{ gh_repo|lower }}
+  name: nbgrader
   version: {{ version }}
 
 source:
-  fn: {{ gh_repo }}-{{ version }}.tar.gz
-  url: https://github.com/{{ gh_org }}/{{ gh_repo }}/archive/{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  url: https://github.com/jupyter/nbgrader/archive/{{ version }}.tar.gz
+  sha256: 971a37b58a8f1a26e7b17eacb091879b5f1646f5097b717626436a266a178d33
 
 build:
-  number: 0
-  skip: true  # [win and py27]
+  number: 1
+  script: python -m pip install --no-deps .
```

Comment:
I am not keen on the details but I do not that modern `pip` does some magic under the hood with the `entry_points`. We (`conda-forge`) should move to that build string as soon as possible to avoid headaches. (I fixed two other feedstocks with the same issue today.)